scripts/scripts.js

```diff
@@ -812,14 +811,18 @@ async function decoratePage(win = window) {
         const usp = new URLSearchParams(window.location.search);
         const martech = usp.get('martech');
 
-        if (!(martech === 'off' || document.querySelector(`head script[src="${martechUrl}"]`))) {
-          let ms = 3000;
-          const delay = usp.get('delay');
-          if (delay) ms = +delay;
-          setTimeout(() => {
+        let ms = 3000;
+        const delay = usp.get('delay');
+        if (delay) ms = +delay;
+        setTimeout(() => {
+          if (!(martech === 'off' || document.querySelector(`head script[src="${martechUrl}"]`))) {
             loadScript(martechUrl, null, 'module');
-          }, ms);
-        }
+          }
+
+          if (document.querySelector('.article-header') && !document.querySelector('[data-origin]')) {
```

Comment:
i think these 4 lines should just go to the bottom of `delayed.js`, and then we can leave the rest off... i think at a certain point we should refactor things to change all the verbiage from `martech` to `delayed` eg. `delayedScript` instead of `martechUrl`